data/sql/updates/pending_db_world/rev_1719238141739725400.sql

```diff
@@ -0,0 +1,4 @@
+-- Raise Dead
+DELETE FROM `spell_cooldown_overrides` WHERE `Id`=34019;
+INSERT INTO `spell_cooldown_overrides` (`Id`, `RecoveryTime`, `CategoryRecoveryTime`, `StartRecoveryTime`, `StartRecoveryCategory`, `Comment`) VALUES
```

Comment:
why do I get a feeling this entire table is a hack.... Also I would prefer this to be in SpellInfoCorrections instead and remove this table